packages/rich-text/src/component/index.js

```diff
@@ -137,10 +139,13 @@ export function useRichText( {
 
 		// Selection must be updated first, so it is recorded in history when
 		// the content change happens.
-		onSelectionChange( start, end );
-		onChange( _value.current, {
-			__unstableFormats: formats,
-			__unstableText: text,
+		// We batch both calls to only attempty to rerender once.
```

Comment:
Nitpick, but there's a typo ("attempty") here.

Reply:
Fixed in trunk, thanks :)